### PR TITLE
Initial commit of arista specifics

### PR DESCRIPTION
**Pull request by kthommandra**
First commit of arista specifics

### CODE REVIEW

File: Arista/ConfigFiles/kapacitor-linux.conf [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+[task]
+  dir = "/persist/kapacitor/tasks"
+  snapshot-interval = "1m0s"
+
+[[influxdb]]
+  enabled = true
+  name = "ServerStats"
+  default = false
+  urls = ["http://influx101:8086"]
```
Comment (subhachandrachandra) [inline Arista/ConfigFiles/kapacitor-linux.conf]: use ServerStats

File: Arista/ConfigFiles/kapacitor-linux.conf [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+[smtp]
+  enabled = true
+  host = "prod-mx.aristanetworks.com"
+  port = 25
+  username = ""
+  password = ""
+  no-verify = false
+  global = false
+  state-changes-only = false
+  from = ""
```
Comment (subhachandrachandra) [inline Arista/ConfigFiles/kapacitor-linux.conf]: Lets use kapacitor@arista.com for from

File: Arista/arista-package.sh [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+LICENSE=MIT
+URL=github.com/aristanetworks/kapacitor
+DESCRIPTION="Time series data processing engine"
+VENDOR=Influxdata
+
+set -e
+
+# Get version from tag closest to HEAD
+COMMIT="e64b52e05dd7c888fe0549a06db3cac118a63dec"
```
Comment (subhachandrachandra) [inline Arista/arista-package.sh:36]: Add comment on why we are choosing a specific version

File: Arista/arista-package.sh [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+for udf in {"withoutUpdates","countle"}
+do
+   cp Ticks/$udf $TMP_CONFIG_DIR/etc/kapacitor/ticks/
+done
+for tick in {"hostdown","disconnects"}
+do
+   cp Ticks/${tick}.tick $TMP_CONFIG_DIR/etc/kapacitor/ticks/
+done
+
+fpm -s dir -t rpm $CONFIG_FPM_ARGS --description "Linux configuration" -n "kapacitor-Linux" etc lib || cleanup_exit 1
```
Comment (subhachandrachandra) [inline Arista/arista-package.sh]: Rename to kapacitor-ServerStats

File: Arista/ConfigFiles/kapacitor-linux.conf (hunk 1 above)
Comment (kthommandra) [inline Arista/ConfigFiles/kapacitor-linux.conf]: done

**Pull request merged**